### ISSUE TITLE
A11y: toast issues

### ISSUE THREAD

**Issue by Fupete** [italia/designers.italia.it#572]
The toast components (the copy action for share and component viewer) has a11y issues.

In an ideal world, a toast like ours should be implemented as an empty and hidden div with `aria-live="polite"` always present on the page, filled at the right moment with the right text via JavaScript. 

We must verify our development, is it like this @bfabio @astagi? 
And, for future reference, @astagi I think later we will need to update the "Notification" component documentation in Bootstrap Italia, to document this specific need for a11y in cases like ours.

**Comment by Fupete** [comment]
Reference: https://www.sarasoueidan.com/blog/accessible-notifications-with-aria-live-regions-part-1/

**Comment by Fupete** [comment]
Alla luce delle nuove verifiche di a11y si riporta che il componente ora funziona correttamente, segnalando ai lettori di schermo la propria presenza e aggiornamento, seppur non seguendo pedissequamente gli aria-pattern standard. Vista la bassa priorità chiudo per adesso la presente in attesa di future implementazioni su Bootstrap Italia cc @astagi .
